torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -591,7 +590,23 @@ def rotate(
 def pad_image_tensor(
     img: torch.Tensor,
     padding: Union[int, List[int]],
-    fill: Optional[Union[int, float]] = 0,
+    fill: Optional[Union[int, float, List[float]]] = None,
+    padding_mode: str = "constant",
+) -> torch.Tensor:
+    if fill is None:
+        # This is a JIT workaround
+        return _pad_with_scalar_fill(img, padding, fill=None, padding_mode=padding_mode)
+    elif isinstance(fill, (int, float)) or len(fill) == 1:
+        fill_number = fill[0] if isinstance(fill, list) else fill
+        return _pad_with_scalar_fill(img, padding, fill=fill_number, padding_mode=padding_mode)
```

Comment:
I just wonder if we could do something like:
```suggestion
    if fill is None or isinstance(fill, (int, float)) or len(fill) == 1:
        if fill is None:
            fill = 0
        fill_number = fill[0] if isinstance(fill, list) else fill
        return _pad_with_scalar_fill(img, padding, fill=fill_number, padding_mode=padding_mode)
```

To avoid the duplicate dispatch to scalar. I assume that `fill-None` ends up becoming 0?

Reply:
I was thinking that below code is valid, but JIT does not agree
```
    if fill is None or isinstance(fill, (int, float)) or len(fill) == 1:
        fill_number = fill[0] if isinstance(fill, list) else fill
```